### DISCUSSION TITLE
Windows 8.1

### DISCUSSION THREAD

**Discussion by T-Sean2** [valinet/ExplorerPatcher#1470] [Ideas]
You should add an option for a windows 8 start screen. I personally find it much nicer then the Windows 10 one and more touch friendly.

I know there is the start screen in windows 10 style but it is different from the one I am asking for.

**Comment by XuanChen1234** [comment]
This is Windows 8.1 start screen.
![desktop-tile-is-back](https://github.com/valinet/ExplorerPatcher/assets/140480110/c70e7c23-4daa-4a00-b8d8-49d16f67f87b)

**Comment by T-Sean2** [comment]
Yes.

**Comment by ajaxnotmeta** [comment]
if redoing windows 8 start screen were impossible
at least it will be convinient to had an option to scroll the tile horizontally instead of vertically like what windows 10 do

**Reply by XuanChen1234** [reply]
You can ask from @valinet and @Amrsatrio

**Comment by pyrates999** [comment]
https://www.thewindowsclub.com/how-to-enable-windows-10-full-screen-start-menu-using-regedit-or-gpedit

**Reply by ajaxnotmeta** [reply]
OP knows the existence of Windows 10 Startscreen or Fulscreen Start
they just want to add emulation of WIndows 8/8.1 Startscreen since this and windows 10 has some minor differences(especially to start tiles being vertical in windows 10 and sideways in 8/8.1)
and also windows 11 now dont have windows 10 start UI anymore so explorer patcher needs to reimplement the features itself

**Reply by pyrates999** [reply]
Unless the code for the windows 8/8.1 start screen is still in windows 11, this is not possible.

**Comment by 4194304** [comment]
I use Linux and this would be an absolute dream come true. It's the only thing that would ever make me go back to Windows

**Comment by XiaoTDM** [comment]
The current Windows 10 Start menu is implemented using code left by Microsoft, which means this Start menu is not customizable. Therefore, it cannot revert to the Windows 8 style unless it is rewrite.
Additionally, in Windows 10 version 1507, it was possible to revert to the Windows 8 style, but unfortunately, it was removed in version 1603.

**Comment by 4194304** [comment]
Since Explorer7 exists, I think the best idea would be to make a fork that focuses on 8.1's explorer instead (Explorer8?). Windows 11 can still run Metro applications, so the core framework is entirely there to run the start screen, which would make it more feasible. Also on Windows 10 1507 (and 1511 I believe as well), you can run the start screen with a little bit of tweaking although it's quite broken. For those who love 8.1's user interface like me, a new project like Explorer8 would be the best option my opinion
